templates/pupil.jade

```diff
@@ -38,9 +38,9 @@ block main
             div.Grid-cell.Grid-cell--full
               div(style="color: white;").Feature
                 h1 Hardware
-                h4 Pupil headsets are 3D printed frames that you wear like a pair of glasses that connect to your device via USB.
-                h4 Cameras record your field of view and your eye movements. Pupil Capture software uses the videos to generate gaze and eye tracking data in real-time.
-                h4 The hardware is designed as a #[strong modular system] that can be configured to support your research and application requirments.
+                h4 The Pupil Headsets are plug and play USB devices carefully designed to be lightweight, unobtrusive and easy to use.
```

Comment:
Consider this revision: `Pupil Headsets are plug and play USB devices carefully designed to be lightweight, unobtrusive, and easy to use.`